src/core/protocol-authorization.ts

```diff
@@ -53,10 +52,10 @@ export class ProtocolAuthorization {
       ancestorMessageChain
     );
 
-    // Verify `dataFormat` and `schema` for the given `recordDefinition`
-    ProtocolAuthorization.verifyRecordDefinition(
+    // Verify `dataFormat` and `schema` for the given `type`
```

Comment:
Nit:
```suggestion
    // Verify `dataFormat` and `schema` for the given `types`
```
or
```suggestion
    // Verify the type of the given message
```
or just remove the comment

Reply:
removed comment